app/views/layouts/_top.html.erb

```diff
@@ -1,25 +1,52 @@
 <div class="top">
   <div id="central-block">
-    <div class="icon"><%= link_to("", root_path) %></div>
-    <div class="menu-links"><%= link_to("Features", "#") %></div>
-    <div class="menu-links"><%= link_to("Business", "#") %></div>
-    <div class="menu-links"><%= link_to("Explore", "#") %></div>
-    <div class="menu-links"><%= link_to("Marketplace", "#") %></div>
-    <div class="menu-links"><%= link_to("Pricing", "#") %></div>
+    <% if logged_out? %>
+      <div class="icon"><%= link_to("", root_path) %></div>
+    <% end %>
+    <% if logged_in?%>
+      <div class="icon"><%= link_to("", current_user) %></div>
+    <% end %>
+    <% if logged_in? %>
```

Comment:
```<% if logged_in?%>``` duplication in rows 6 and 9.
Also, can we move logged_in and logged_out code blocks into partials?

Reply:
Looked on the code from another side and understood that I don't need logged_out method in here.